resources/fetching/factory/eks_factory.go

```diff
@@ -27,7 +27,8 @@ import (
 
 	"github.com/elastic/cloudbeat/resources/fetching"
 	"github.com/elastic/cloudbeat/resources/fetching/condition"
-	"github.com/elastic/cloudbeat/resources/fetching/fetchers"
+	awsFetchers "github.com/elastic/cloudbeat/resources/fetching/fetchers/aws"
+	k8sFetcher "github.com/elastic/cloudbeat/resources/fetching/fetchers/k8s"
```

Comment:
nit: is it fetcher or fetchers? nit2: in any case I think the convention is to avoid capitalization altogether so it should be something like `k8sfetcher` [see here](https://go.dev/blog/package-names)